### ISSUE TITLE
Travis fails to install dep

### ISSUE THREAD

**Issue by jdsutherland** [exercism/cli#781]
Travis is failing to install `dep`. See [Travis job](https://travis-ci.org/exercism/cli/jobs/467827450).
```
$ go get -u github.com/golang/dep/...
package github.com/golang/dep/...: github.com/golang/dep/...: invalid import path: malformed import path "github.com/golang/dep/...": double dot
The command "go get -u github.com/golang/dep/..." failed and exited with 1 during .
```

**Comment by kytrinyx** [comment]
Hey @jdsutherland, I'm so sorry I went radio silent for the past month. I'm back now, and will be working my way through the issues and PRs in this repo over the next few days.

**Comment by ZapAnton** [comment]
@jdsutherland, could this can be closed, since #812 was merged?
